http/lib.c

```diff
@@ -114,6 +114,9 @@ lbox_httpd_escape_html(struct lua_State *L)
 			continue;
 		}
 		const char *s = lua_tostring(L, i);
+		if (s == NULL) {
```

Comment:
Thanks for the investigation.

I think the fix is not quite correct. We should do something like tostring() for a returned value, not just ignore it. Please, consider copying of luaT_tolstring() into lib.c source code.

https://github.com/tarantool/tarantool/commit/01c3270117b337cb8f1c0c1309d3ae7f46f9a948

Reply:
This function lead to CI fail (and it's quite strange) - I've adopted it

Reply:
Maybe due to changing of the Lua stack top. Anyway, I left a comment regarding this, so this one can be resolved.